src/domain/community/organization-verification/organization.verification.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { Column, Entity, JoinColumn, OneToOne } from 'typeorm';
+import { IOrganizationVerification } from './organization.verification.interface';
+import { OrganizationVerificationEnum } from '@common/enums/organization.verification';
+import { Lifecycle } from '@domain/common/lifecycle/lifecycle.entity';
+import { AuthorizableEntity } from '@domain/common/entity/authorizable-entity';
+
+@Entity('OrganizationVerification')
```

Comment:
Specifying the name of the entity as a string when the class name is the same is redundant - remove it.